modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -1004,19 +1019,30 @@ type GmosNorthStatic {
   nodAndShuffle: GmosNodAndShuffle
 }
 
-# GMOS North static configuration input parameters
+"""
+GMOS North static configuration input parameters
+"""
 input GmosNorthStaticInput {
-  # GMOS North Detector option
-  detector: GmosNorthDetector! = HAMAMATSU
+  """
+  GMOS North Stage Mode (default to FOLLOW_XY)
+  """
+  stageMode: GmosNorthStageMode
```

Comment:
I wanted to make this:

```
stageMode: GmosNorthStageMode = FOLLOW_XY
```

but Grackle can't seem to handle supplying the default value so I settled for supplying it in code and documenting it here.  :-/